### ISSUE TITLE
adding hydrogens ignores the pH option

### ISSUE THREAD

**Issue by amjjbonvin** [bioexcel/biobb_chemistry#6]
**Describe the bug**

When running the ligand parametrisation example and part adding the hydrogens, obabel reports that it will ignore the defined pH because both the `-p` and `-p` options are given. Extract from a log file:

```
*** Open Babel Warning  in DoTransformations
  Both -p and -h options are set. All implicit hydrogens (-h) will be added without considering pH.
```

**To Reproduce**

Run the ligand parametrisation notebook, and check the log file generated by the `Add Hydrogens` step

**Expected behavior**

the `-h` and `-p` options should be exclusive, allowing to control the pH to define the protonation state.

**Desktop (please complete the following information):**
 - OS: OSX 13.6.2
 - Python version: 3.12.0 (installed via miniconda)
 - BioBB version 4.1.0

**Comment by amjjbonvin** [comment]
Tested manually with obabel that when using only the `-p` options hydrogens are added considering the defined pH. With both `-h` and `-p` all hydrogens are added by default, irrespective of the defined pH

**Comment by gbayarri** [comment]
Fixed in PR https://github.com/bioexcel/biobb_chemistry/pull/7, it will be available in the next release.

Thanks!

**Comment by amjjbonvin** [comment]
This strange to close an issue as completed when nothing has happened… it is not because it was stale that it is completed.

**Comment by gbayarri** [comment]
Hi, @amjjbonvin , it's an automatic action. It's reopened until next release.

Best.

**Comment by amjjbonvin** [comment]
Hi Genis

Thanks.
Then you could close it as “not planned” or something else, but not as “completed” (might be easier to extract stats of what has been done) :-)

Cheers
Alexandre

**Comment by gbayarri** [comment]
Fixed in the new [4.2.0 version](https://anaconda.org/bioconda/biobb_chemistry)
